uamqp/utils.py

```diff
@@ -93,7 +94,44 @@ def add_batch(batch, message):
     batch.data.append(output)
 
 
+def encode_str(data, encoding='utf-8'):
```

Comment:
Not sure we need this if we're no longer supporting 2.7

Reply:
This is only called by `normalized_data_body` to encode string to bytes as data body only takes bytes or list of bytes.